### DISCUSSION TITLE
Pre-Flight & Setup Issue

### DISCUSSION THREAD

**Discussion by MynameisStevetoo** [snipe/snipe-it#14761] [Q&A]
Hi all, 

So I have followed the process at every step so far, but for some reason, I can't get the pre-flight to kick in. When I navigate to the page, nothing happens. I have check my .htaccess, my URL in the .env file, mod_rewrite is working ok. All the dependencies etc are installed; So I'm feeling a little at a loss, I'm hoping it's something incredibly simple I have overlooked! :D 

Any advice at all?

Thanks

**Comment by marcusmoore** [comment]
What exactly happens when you navigate to the application? Does it land on a blank page? Do you have any entries in apache's access log?

**Reply by MynameisStevetoo** [reply]
Thanks for the reply Marcus! It lands on the hosting company "Coming soon" default page. if I stick an index file in there it loads fine. Checked the access logs, no issues there, for today so far:

```
XXXXXXXXXXXX - - [24/May/2024:00:39:57 -0700] "GET / HTTP/2.0" 200 662 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36" 
XXXXXXXXXXXX - - [24/May/2024:00:44:51 -0700] "GET / HTTP/2.0" 200 662 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36" 
XXXXXXXXXXXX - - [24/May/2024:00:44:52 -0700] "GET /favicon.ico HTTP/2.0" 200 269 "https://XXXXXXXXXXXX.XXXXXXXXXXXX.net/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36"
```

**Reply by marcusmoore** [reply]
Hm...It seems like the server configuration is not pointing to the application? You'll want to make sure apache is set to the `public` directory of wherever the application is installed. Could that be it?

More details: https://snipe-it.readme.io/docs/linuxosx
